octobot_trading/api/symbol_data.py

```diff
@@ -87,3 +89,9 @@ def force_set_mark_price(exchange_manager, symbol, price):
 def is_mark_price_initialized(exchange_manager, symbol: str) -> bool:
     return exchange_manager.exchange_symbols_data.get_exchange_symbol_data(symbol).prices_manager.\
         valid_price_received_event.is_set()
+
+
+async def subscribe_to_ohlcv_channel(callback, exchange_id):
```

Comment:
I don't understand why it's in symbol_data api

Reply:
It's data related to a traded symbol, where do you see it otherwise ?

Reply:
I'm adding a channels.py api